src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -97,7 +97,7 @@ impl<'a> Resolver<'a> {
 
     crate fn get_module(&mut self, def_id: DefId) -> Module<'a> {
         if def_id.krate == LOCAL_CRATE {
-            return self.module_map[&def_id];
+            return self.module_map[&def_id.as_local().unwrap()];
```

Comment:
This code pattern should be changed to `if let Some(def_id) = def_id.as_local() {`.